package.json

```diff
@@ -48,13 +48,17 @@
     "jshint": "jshint --show-non-errors ."
   },
   "dependencies": {
-    "sax": "~1.2.1",
     "coa": "~1.0.1",
-    "js-yaml": "~3.7.0",
     "colors": "~1.1.2",
-    "whet.extend": "~0.9.9",
+    "css-select": "^1.3.0-rc0",
+    "css-select-base-adapter": "^0.1.0",
+    "csso": "~2.3.1",
```

Comment:
Just so you know, stuff seems to break when you upgrade to the most recent version of csso (v3.0.0). I get the following error:

ERROR in ./src/app/scripts/svgo/plugins/inlineStyles.ts
Module not found: Error: Can't resolve 'csso/lib/compressor/restructure/prepare/specificity' in '/Users/alockwood/ShapeShifter/src/app/scripts/svgo/plugins'
 @ ./src/app/scripts/svgo/plugins/inlineStyles.ts 6:20-82